.gitignore

```diff
@@ -1,27 +1,106 @@
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
```

Comment:
Dulwich doesn't generate these files.

Reply:
This is Jpython for Java, and it is also your own original ignore list. I just moved the position. Many of them are originally there, but you said you want to delete it.